bin/tflocal

```diff
@@ -60,6 +60,7 @@ terraform {
     bucket         = "<bucket>"
     key            = "<key>"
     dynamodb_table = "<dynamodb_table>"
+    use_path_style = "<use_path_style>"
```

Comment:
This must be added dynamically, see my other comment below why.

Reply:
To keep transparency and a similar style, I left it here, but changed the default to false.